Libraries/PeriphDrivers/Include/MAX32665/icc.h

```diff
@@ -85,6 +85,32 @@ void MXC_ICC_Disable(void);
  */
 void MXC_ICC_Flush(void);
 
+/**
+ * @brief   Reads ID information from one of the ICC's Cache ID Register.
+ * @param   icc     Pointer ICC instance to get ID information from.
+ * @param   cid     Selects what information to get from the Cache ID Register
+ * @retval  Returns the selected value from the Cache ID Register.
+ */
+int MXC_ICC_IDInst(mxc_icc_regs_t *icc, mxc_icc_info_t cid);
```

Comment:
Consider...
```C
int MXC_ICC_GetID(mxc_icc_regs_t *icc, mxc_icc_info_t cid);
````
instead.  The other function name `MXC_ICC_ID` is terrible so in this case we should make this one the standard.

Reply:
I agree that that's a much better name, but I want it to be consistent with what's in MAX32650/icc.h. We could deprecate it? It's only been around for a month or two.

Reply:
Ah - thought you were creating a new function.  If `MXC_ICC_IDInst` already exists then we're forced to use it.  No change needed.